packages/coinstac-api-server/test/test-setup.js

```diff
@@ -67,14 +68,13 @@ helperFunctions.getRethinkConnection()
         controller: { type: 'decentralized' },
         id: 'HyLfdfanb',
         inputMap: {
-          covariates: [
-            {
-              name: 'biased xs',
-              source: { inputKey: 'biasedX', inputLabel: 'Computation 1: Biased Xs', pipelineIndex: 0 },
-              type: 'number',
-            },
-          ],
-          freeSurferRegion: { value: ['5th-Ventricle'] },
+          covariates: {
+            ownerMappings: [
+              {
+                fromCache: { variable: 'biasedX', step: 0 },
```

Comment:
Interesting, I didn't quite think of this and wonder if it'll be used, but see no harm in it.

Reply:
Yeah, for now it just shows the link in the UI, but the case isn't really handled when data is passed off to pipeline. I figured I'd leave it there as an idea for now if we want to return to it.